File: Model/Structure.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Model
{
    [Table("structure")]
    public class Structure
    {
        public const string PROFIL_STRUCTURE = "STRUCTURE";

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("id")]
        public int? Id { set; get; }
        [Column("email")]
        public string Email { set; get; }
        [Column("password")]
        public string Password { set; get; }
        [Column("profil")]
        public string Profil { set; get; }
        [Column("street")]
        public string Street { set; get; }
        [Column("city")]
        public string City { set; get; }
        [Column("post_code")]
        public string PostCode { set; get; }
        [Column("department")]
        public string Department { set; get; }
        [Column("state")]
        public string State { set; get; }
        [Column("country")]
        public string Country { set; get; }
        [Column("phone")]
        public int? Phone { set; get; }
        [Column("longitude")]
        public string Longitude { set; get; }
        [Column("latitude")]
        public string Latitude { set; get; }
        [Column("siret")]
        public string Siret { set; get; }
        [Column("name")]
        public string Name { set; get; }

        [NotMapped]
        public string ConfirmPassword { get; set; }
    }
}

File: WebAPP/ClientApp/src/dto/structure.ts
export class Structure {
	public id: number;
	public email: string;
	public password: string;
	public profil: string;
	public street: string;
	public city: string;
	public postCode: string;
	public department: string;
	public state: string;
	public country: string;
	public phone: number;
	public longitude: string;
	public latitude: string;
	public siret: string;
	public name: string;
	public confirmPassword: string;

	public static structureProfil = "STRUCTURE";

	constructor(id?: number, email?: string, password?: string, profil?: string, street?: string, city?: string,
		postCode?: string, department?: string, state?: string, country?: string,  phone?: number, longitude?: string,
		latitude?: string, siret?: string, name?: string, confirmPassword?: string)

	constructor(id: number, email: string, password: string, profil: string, street: string, city: string,
		postCode: string, department: string, state: string, country: string, phone: number,
		longitude: string, latitude: string, siret: string, name: string, confirmPassword: string) {
		this.id = id;
		this.email = email;
		this.password = password;
		this.profil = profil;
		this.street = street;
		this.city = city;
		this.postCode = postCode;
		this.department = department;
		this.state = state;
		this.country = country;
		this.phone = phone;
		this.longitude = longitude;
		this.latitude = latitude;
		this.siret = siret;
		this.name = name;
		this.confirmPassword = confirmPassword;
	}
}
